runtimes/eden/src/xcm_config.rs

```diff
@@ -0,0 +1,128 @@
+use super::{AccountId, Balance, Call, Event, Origin, ParachainSystem, PolkadotXcm, Runtime, ParachainInfo, XcmpQueue, Balances};
+use frame_support::{
+	parameter_types,
+	traits::{Everything, Nothing},
+	weights::{IdentityFee},
+};
+use frame_system::EnsureRoot;
+use polkadot_parachain::primitives::Sibling;
+use pallet_xcm::XcmPassthrough;
+use xcm::{latest::Weight as XcmWeight,latest::NetworkId, prelude::*};
+use xcm_builder::{
+	AllowKnownQueryResponses, AllowSubscriptionsFrom, AllowTopLevelPaidExecutionFrom, EnsureXcmOrigin,
+	FixedWeightBounds, LocationInverter, NativeAsset, RelayChainAsNative, SiblingParachainAsNative,
+	SignedAccountId32AsNative, SignedToAccountId32, TakeWeightCredit, UsingComponents, SovereignSignedViaLocation, AccountId32Aliases, SiblingParachainConvertsVia, ParentIsPreset,
+};
+use xcm_executor::{Config, XcmExecutor};
+
+/// Type for specifying how a `MultiLocation` can be converted into an `AccountId`. This is used
+/// when determining ownership of accounts for asset transacting and when attempting to use XCM
+/// `Transact` in order to determine the dispatch RuntimeOrigin.
+pub type LocationToAccountId = (
+	// The parent (Relay-chain) origin converts to the default `AccountId`.
+	ParentIsPreset<AccountId>,
+	// Sibling parachain origins convert to AccountId via the `ParaId::into`.
+	SiblingParachainConvertsVia<Sibling, AccountId>,
+	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
+	AccountId32Aliases<RelayNetwork, AccountId>,
+);
+
+pub type Barrier = (
+	TakeWeightCredit,
+	// If the message is one that immediately attemps to pay for execution, then allow it.
+	AllowTopLevelPaidExecutionFrom<Everything>,
+	// Expected responses are OK.
+	AllowKnownQueryResponses<PolkadotXcm>,
+	// Subscriptions for version tracking are OK.
+	AllowSubscriptionsFrom<Everything>,
+);
+
+pub type LocalOriginToLocation = SignedToAccountId32<Origin, AccountId, RelayNetwork>;
+
+/// The means for routing XCM messages which are not for local execution into the right message
+/// queues.
+pub type XcmRouter = (
+	// Two routers - use UMP to communicate with the relay chain:
+	cumulus_primitives_utility::ParentAsUmp<ParachainSystem, PolkadotXcm>,
+	// ..and XCMP to communicate with the sibling chains.
+	XcmpQueue,
+);
+parameter_types! {
+	pub DotLocation: MultiLocation = MultiLocation::parent();
```

Comment:
If we go for `NetworkId::Any` as RelayNetwork, then this is better named RelayLocation. This is because the token of the relay chain is not necessarily called Dot.

Reply:
done